community/tools/graphql2chartjs/bundle/js/index.min.js

```diff
@@ -0,0 +1,2029 @@
+(function (global, factory) {
```

Comment:
This file is not minified. Also, remove this from the repo and put it on some CDN.